### PR TITLE
Rename Phonecontrol to Devicecontrol

### PR DESCRIPTION
**Pull request by muckelba**
retry of #311 

Phone isn't the right name anymore :D I also changed some loglines, so its not only MADmin

### CODE REVIEW

File: madmin/routes/control.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def get_phonescreens(self):
                                                 dummy=True)
                             )
 
-        return render_template('phonescreens.html', editform=screens_phone, header="Phonecontrol", title="Phonecontrol")
+        return render_template('phonescreens.html', editform=screens_phone, header="Device scontrol", title="Device control")
```
Comment (sn0opy) [inline madmin/routes/control.py]:
```suggestion
        return render_template('phonescreens.html', editform=screens_phone, header="Device control", title="Device control")
```

File: madmin/routes/control.py [hunk 2]
```diff
@@ -324,15 +324,15 @@ def clear_game_data(self):
         devicemappings = self._mapping_manager.get_all_devicemappings()
 
         adb = devicemappings.get(origin, {}).get('adb', False)
-        self._logger.info('MADmin: Clear game data for phone ({})', str(origin))
+        self._logger.info('MADmin: Clear game data for Device ({})', str(origin))
```
Comment (sn0opy) [inline madmin/routes/control.py]:
```suggestion
        self._logger.info('MADmin: Clear game data for device ({})', str(origin))
```

File: madmin/static/vars/template/phone.tpl [hunk 3]
```diff
@@ -13,7 +13,7 @@
 <div id=button><a id=screenshot origin=<<phonename>> href='take_screenshot?origin=<<phonename>>&adb=<<adb_option>>'>Take Screenshot</a></div>
 <div id=button><a href='clear_game_data?origin=<<phonename>>&adb=<<adb_option>>' id='resetgamedata' origin=<<phonename>> class='confirm' title='Do you really want to clear data for pogo, this will result in the need to re-login?'>Reset Game Data</a></div>
 <div id=button><a href='quit_pogo?origin=<<phonename>>&adb=<<adb_option>>' id='quit' origin=<<phonename>>>Quit Pogo</a></div>
-<div id=button><a href='restart_phone?origin=<<phonename>>&adb=<<adb_option>>' class='confirm' title='Do you really want to restart phone?'>Reboot Phone</a></div>
+<div id=button><a href='restart_phone?origin=<<phonename>>&adb=<<adb_option>>' class='confirm' title='Do you really want to restart the device?'>Reboot Device</a></div>
```
Comment (sn0opy) [inline madmin/static/vars/template/phone.tpl]:
```suggestion
<div id=button><a href='restart_phone?origin=<<phonename>>&adb=<<adb_option>>' class='confirm' title='Do you really want to restart the device?'>Reboot device</a></div>
```

File: utils/walkerArgs.py [hunk 4]
```diff
@@ -278,7 +278,7 @@ def parseArgs():
 
     # adb
     parser.add_argument('-adb', '--use_adb', action='store_true', default=False,
-                        help='Use ADB for phonecontrol (Default: False)')
+                        help='Use ADB for Device control (Default: False)')
```
Comment (sn0opy) [inline utils/walkerArgs.py]:
```suggestion
                        help='Use ADB for "device control" (Default: False)')
```

File: worker/WorkerBase.py [hunk 5]
```diff
@@ -587,7 +587,7 @@ def _check_windows(self):
                                                    ScreenType.PERMISSION)) \
                         and self._last_screen_type == returncode \
                         and self._same_screen_count == 3:
-                    logger.warning('Pogo freeze - restart Phone')
+                    logger.warning('Pogo freeze - restart Device')
```
Comment (sn0opy) [inline worker/WorkerBase.py]:
```suggestion
                    logger.warning('Gamefreezed - restarting device')
```

File: worker/WorkerBase.py [hunk 6]
```diff
@@ -626,7 +626,7 @@ def _check_windows(self):
                     self._loginerrorcounter += 1
 
                 elif returncode == ScreenType.GPS:
-                    logger.warning("Detecting GPS Error 11 - reboot phone")
+                    logger.warning("Detecting GPS Error 11 - reboot Device")
```
Comment (sn0opy) [inline worker/WorkerBase.py]:
```suggestion
                    logger.warning("Detected GPS error 11 - rebooting device")
```

File: worker/WorkerBase.py [hunk 7]
```diff
@@ -642,7 +642,7 @@ def _check_windows(self):
                     break
 
                 if self._loginerrorcounter == 2:
-                    logger.error('Cannot login again - (clear pogo game data and) restart phone')
+                    logger.error('Cannot login again - (clear pogo game data and) restart Device')
```
Comment (sn0opy) [inline worker/WorkerBase.py]:
```suggestion
                    logger.error('Couldn't login again - (clearing game data + restarting device')
```

File: configs/config.ini.example [hunk 8]
```diff
@@ -81,7 +81,7 @@
 
 # ADB
 ######################
-#use_adb                      # Use ADB for phonecontrol (Default: False)
+#use_adb                      # Use ADB for Device control (Default: False)
```
Comment (sn0opy) [inline configs/config.ini.example]:
```suggestion
#use_adb                      # Use ADB for "device control" (Default: False)
```

File: madmin/routes/control.py [hunk 9]
```diff
@@ -307,15 +307,15 @@ def restart_phone(self):
         devicemappings = self._mapping_manager.get_all_devicemappings()
 
         adb = devicemappings.get(origin, {}).get('adb', False)
-        self._logger.info('MADmin: Restart Phone ({})', str(origin))
+        self._logger.info('MADmin: Restart Device ({})', str(origin))
```
Comment (sn0opy) [inline madmin/routes/control.py]:
```suggestion
        self._logger.info('MADmin: Restart device ({})', str(origin))
```

**Comment by muckelba** [comment]
Ready to merge :)

**Pull request merged**